Assets/Scripts/Managers/HealthDisplayManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Assets.Scripts.Managers
+{
+    class HealthDisplayManager : MonoBehaviour
+    {
+        public static HealthDisplayManager Instance { get; private set; } //Singleton
+
+        Text fraction; //Number display on the bars
+        Transform under; //The bar you see under
+        Transform over; //The bar you see over
+
+        PlayerManager playerManager; //Reference to player
+
+        // Use this for initialization
+        void Start()
+        {
+            Instance = this;
+            this.fraction = GetComponentInChildren<Text>();
+            this.under = GetComponentsInChildren<Image>()[0].transform;
+            this.over = GetComponentsInChildren<Image>()[1].transform;
```

Comment:
It would be better to import these as ```public``` or as ```[SerializeField] private``` as GetComponent is slow and dependent on the order of the children.